src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -464,15 +470,23 @@ private boolean tryToLaunchNameNode(SchedulerDriver driver, Offer offer) {
     } else if (deadNameNodes.contains(offer.getHostname())) {
       launch = true;
     }
-    if (launch) {
-      return launchNode(
-          driver,
-          offer,
-          HDFSConstants.NAME_NODE_ID,
-          Arrays.asList(HDFSConstants.NAME_NODE_ID, HDFSConstants.ZKFC_NODE_ID),
-          HDFSConstants.NAME_NODE_EXECUTOR_ID);
+    if (!launch) {
+      return false;
     }
-    return false;
+
+    String nameNodeTaskName = getNextTaskName(HDFSConstants.NAME_NODE_ID);
+    if (nameNodeTaskName.isEmpty()) return false;
```

Comment:
To be consistent with our style here, we need to use brackets and put return false on the next line. We want to keep our style consistent throughout the project ideally.